Makefile

```
@@ -0,0 +1,43 @@
+.PHONY: all
+all: build-coredns
+
+GO_TEST_FLAGS = $(VERBOSE)
```

That looks misleading to me. `$VERBOSE` should control verbosity of something, here it controls the flags of `go test`.  Would something like this do what you wanted?
```suggestion
GO_TEST_FLAGS ?= -v
```

---

I'll make the change.